src/engine/guidance/post_processing.cpp

```diff
@@ -198,6 +198,7 @@ std::vector<std::vector<PathData>> postProcess(std::vector<std::vector<PathData>
     // silence silent turns for good
     for (auto &path_data : leg_data)
```

Comment:
https://github.com/Project-OSRM/osrm-backend/blob/rewrite/new-api/include/engine/guidance/assemble_leg.hpp#L141-L142

It removes the time value from all non-turn items